packages/back-end/src/events/handlers/slack/slack-event-handler-utils.ts

```diff
@@ -288,6 +294,11 @@ const buildSlackMessageForFeatureDeletedEvent = async (
 export const getExperimentUrlFormatted = (experimentId: string): string =>
   `\n• <${APP_ORIGIN}/experiment/${experimentId}|View Experiment>`;
 
+export const getExperimentUrlAndNameFormatted = (
+  experimentId: string,
+  experimentName: string
+): string => `<${APP_ORIGIN}/experiment/${experimentId}|${experimentName}>`;
```

Comment:
`APP_ORIGIN` was missing indeed! Also didn't realized this was not a `markdown` formatted string..